generator/src/test/java/com/scottlogic/deg/generator/cucumber/features/operators/grammatical/If.feature

```diff
@@ -2435,7 +2435,7 @@ Feature: Values can be specified by using if, then and else constraints
       | foo | bar |
       | 10  | 1   |
 
-  @ignore #issue 363  maybe also 756?
+  @ignore #issue 363
```

Comment:
why is this one still ignored?

Reply:
It produces [1, 2.2], which it shouldn't

Reply:
is this a new bug, or something to do with 363?

Reply:
Related to 363